package.json

```diff
@@ -87,8 +87,8 @@
     "source-map-support": "^0.5.21",
     "standard-version": "^9.5.0",
     "ts-node": "^10.9.1",
-    "tsd": "^0.24.1",
-    "typescript": "^4.8.4",
+    "tsd": "^0.25.0",
+    "typescript": "^4.9.0",
```

Comment:
Ty for pulling this in, can we use 4.9.4 here? I think it's a good idea to pull in the eslint+TS tooling as well when we bump TS. I checked locally, all should pass

```txt
@typescript-eslint/eslint-plugin  ^5.40.0  →  ^5.48.0
@typescript-eslint/parser         ^5.40.0  →  ^5.48.0
eslint                            ^8.25.0  →  ^8.31.0
typescript                         ^4.9.0  →   ^4.9.4
 ```